Add the parameter `out` to `opq` to select the level of verbosity of the results

jmaspons opened the pull request:
I can replicate the option for `opq_osm_id`, `opq_enclosing` and `opq_around`

jmaspons commented:
Checks fail due to a link in the documentation to `osmdata_data_frame` which is not merged yet #285

jmaspons commented:
From my side, ready for the merge @mpadge 

P.S.: Don't know why there are no test coverage checks from the gh actions anymore

mpadge commented on R/opq.R:
```diff
@@ -76,17 +88,30 @@
+    }
+    has_geometry <- out %in% c ("body", "meta", "skel")
+    if (has_geometry && out != "body") {
+        suffix <- gsub ("out( body)*;", paste0 ("out ", out, ";"), suffix)
```
```suggestion
        suffix <- gsub ("out(\\sbody)*;", paste0 ("out ", out, ";"), suffix)
```

mpadge commented on R/opq.R:
```diff
@@ -76,17 +88,30 @@
+    if (has_geometry && out != "body") {
+        suffix <- gsub ("out( body)*;", paste0 ("out ", out, ";"), suffix)
+    } else if (!has_geometry) {
+        suffix <- gsub ("\\);(\\n\\(\\._;>;\\);\\n| )out( body)*;", paste0 ("); out ", out, ";"), suffix)
```
Why not just redefine `suffix` completely?
```suggestion
        suffix <- paste0 ("); out ", out, ";")
```

jmaspons commented:
Thanks for the review. I have rewritten this part by testing the condition for `has_geometry` adding `nodes_only`, and build the suffix from scratch. This makes the tests fails because they expect that queries with nodes_only have no "body" in the string, which is the default anyway. This limit queries with `nodes_only `and out meta;. I have changed the tests
